### PR TITLE
add st price data config on proj level

### PR DESCRIPTION
**Pull request by jacobvjk**
closes https://github.com/2DegreesInvesting/PACTA_analysis/issues/369
blocked by https://github.com/2DegreesInvesting/StressTestingModelDev/pull/180

This PR:
- adds a new config field to the `ProjectParameters_CODE.yml` files
- the field structure is `price_data_verson: YYYYQQ` and it is a sub fiel of `stress_test`
- `set_project_parameters()` writes the new config field to a character variable in the `GlobalEnv` (as it does with all other configs)
- as such, the variable is implicitly passed to the stress test when calling `source(file.path(stress_test_path, "web_tool_stress_test.R"))`
- if no such variable is passed for any reason, the stress test script has a default to fall back to (2020Q4 to ensure the swiss project does not change)
- For CHPA2020, I set the previously existing price data version: 2020Q4
- For PA2020FL, I set the price data version: 2020Q4
- For PA2020GENERAL, I set the new price data version 2021Q1, with re-interpolated prices
- For future projects I strongly suggest to always use 2021Q1 (or newer versions)

### CODE REVIEW

**Review by Clare2D — APPROVED** [review]
LGTM

**Comment by jacobvjk** [comment]
I can only merge this PR, once the corresponding PR in the ST repo is approved, see description at the top :)

**Pull request merged**
